website/blog/2020-07-20-docs-update.md

```diff
@@ -0,0 +1,36 @@
+author: Rachel Nabors
+authorTitle: Documentation Engineer at Facebook
+authorURL: https://twitter.com/rachelnabors
+authorImageURL: https://avatars0.githubusercontent.com/u/236306?s=460&v=4
+authorTwitter: rachelnabors
+---
+
+Last year we conducted user interviews and sent out [a survey](https://www.surveymonkey.co.uk/r/DDZWQDJ) to learn more about how and when people use these React Native docs. With the data and guidance gleaned from 24 interviews and over 3000 survey responses, we've been able to work to improve React Native's documentation, and we're excited to share that progress today:
+
+- **[New Getting Started guides](https://fb.workplace.com/groups/529650227572836/permalink/656344974903360/)** We launched new Getting Started with React Native docs to [explain what Native Components](https://reactnative.dev/docs/intro-react-native-components) are to people with no mobile development background. We also included a [refresher/introduction to React](https://reactnative.dev/docs/intro-react) to help folks getting started with React for the first time!
+- **[New Testing guide](https://reactnative.dev/docs/testing-overview)** We worked with Vojtech Novak to create a new illustrated testing guide that introduces app developers to different kinds of testing strategies and how they work in a React Native workflow.
+- **[New Security guide](https://reactnative.dev/docs/security)** We worked with Kadi Kraman to create a new illustrated security guide that explains the basics of security in a React Native world and puts forth best in class solutions.
+- **More illustrations** including the new [`Pressable`](https://reactnative.dev/docs/pressable) and [introduction to React Native components](https://reactnative.dev/docs/intro-react-native-components) docs
+- **Updated Core Component and API docs** We held a [documentation drive](https://github.com/facebook/react-native-website/issues/1579) to update reference docs! Thanks to these and other participants we were able to fully update the docs and add Snack examples to all of them in time for 0.62: [Marta Dabrowka](https://twitter.com/martadabrowka), [Abraham Nnaji](https://twitter.com/nnajiabraham), [Ahmed Talaat El-Hager](https://twitter.com/ahmdtalat), [Mohamed Abdel Nasser Abdou](https://twitter.com/mohamedsgap), [Danilo Britto](https://twitter.com/danilobrinu), [Mitul Savani](https://twitter.com/mitulsavani), [Kaio Duarte](https://twitter.com/kaiodduarte), [Pablo Espinosa](https://twitter.com/espipj), [Jesse Katsumata](https://twitter.com/natural_clar), [I Gede Agastya Darma Laksana](https://twitter.com/gedeagas), [Sebastião Bruno Kiafuka Fernando](https://twitter.com/bruno_kiafuka), [Miguel Bolivar](https://twitter.com/Darking360), [Dani Akash](https://twitter.com/dani_akash_), [Luiz Celso de Faria Alves](https://twitter.com/_eucelso), and [Bartosz Kaszubowski](https://twitter.com/simek). With their contributions, these are the best and most up to date React Native docs yet!
```

Comment:
What do you think of moving the contributor list to the end of this section on line 22 so that the bullet point shape is more consistent? As is, there's a large paragraph in the middle of shorter bullet points.